doc/formats.md

```diff
@@ -2,126 +2,127 @@
 
 [fq -rn -L . 'include "formats"; formats_table']: sh-start
 
```

Comment:
Now i see why it still caused conflict, apple_bookmark is the new longest format name :)